internal/test/params.go

```diff
@@ -0,0 +1,13 @@
+package test
```

Comment:
this was in `internal/test/factory/`, if that doesn't exist, then here is fine, but leaving this comment here in case this was accidentally not moved to the proper path.

Reply:
Yeah, for some reason, code belonging to `internal/test/factory/` started landing in `internal/test/` last summer. So this is keeping up with that.
I think we'll need to re-think how to rearrange the code currently in `internal/`, since we no longer have "internal" production code. Maybe the way forward is to move again part of the production code to internal. Time will tell.